### ISSUE TITLE
Crash to desktop

### ISSUE THREAD

**Issue by staticlando** [Darkosto/SevTech-Ages#1905]
Semi-random crash to desktop that happens when I'm near my Tinkers smeltery and a squeezer/fermenter automated setup.

Minecraft, OS and Java info included with Pastebin link.

https://pastebin.com/Fi6w6CAR

**Comment by SteseDav** [comment]
Would be helpful to add a brief description of what you are doing in that automated setup, and what other things you feel are helpful in replicating the problem.

The crash report alone may not be enough without context.

**Comment by SteseDav** [comment]
Would also be a good idea to update your java version to the latest (162), and force Minecraft to use it. 

https://docs.google.com/document/d/e/2PACX-1vR5muyi1m7Zy_qSwrzeb9e3tbbRlksguNUb_Pvm3pn9eDpsIkUj5qQUq6WFPPdEoVWZq91WvvH57KzC/pub

Regards,

**Comment by staticlando** [comment]
The automation is four lines (2 of sugarcane, 2 of industrial hemp) with pistons set up on 3 redstone clocks. Material floats down the water to conveyors where I load them into the fermenter and squeezer. If the game lets me in I can grab screenshots if needed.

I also just realized that this didn't start happening until I unlocked journey map.

**Comment by SteseDav** [comment]
Have a go with the above, and see what happens :)

**Comment by staticlando** [comment]
Okay, so now I crashed out of the world to the Minecraft menu, reloaded the world and crashed to desktop. I reloaded the instance, loaded world and crashed to menu again. Then I loaded the world and am able to play.

Technology is great ;) So far it's been letting me play fine for the last 10-15 minutes. I'll keep monitoring the situation and notice if I do anything in particular that sets it off.
